app/uk/gov/hmrc/hec/services/scheduleService/HecTaxCheckExtractionService.scala

```diff
@@ -167,22 +175,56 @@ class HecTaxCheckExtractionServiceImpl @Inject() (
 
   }
 
-  //Combining the process of creating and Storing file
+  //Combining the process of creating , Storing file ,
```

Comment:
it'd be worth renaming this `def` to reflect this info - maybe something like `createAndStoreFileAndNotify`